### PR TITLE
Add idtokens to manual_glidein_submit

### PR DESCRIPTION
**Pull request by mmascher**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,24 @@ SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
...
+
+### Changed defaults / behaviours
+
+### Deprecated / removed options and commands
+
+### Security Related Fixes
+
+### Bug Fixes
+
+-   manual_glidein_submit now correctly sets idtokens in the EncryptedInputFiles (PR #280)
```
Comment (mambelli) [inline CHANGELOG.md:18]: It is `(issue #283)` or `(issue #283, PR#284)` 
Then you can rebase and add it to the 3.10.2 bug fixes to include it in 3.10.2

File: factory/tools/manual_glidein_submit.py [hunk 2]
```diff
@@ -209,6 +209,16 @@ def main():
                 % (scitoken_file, str(credentials.identity_credentials))
             )
 
+        condortoken = "credential_{}_{}.idtoken".format(ad_gc["ClientName"], entry_name)
+        condortoken_file = os.path.join(credentials.cred_dir, condortoken)
+        if not os.path.exists(condortoken_file):
+            logging.warning("Cannot find idtoken file %s" % condortoken_file)
+        elif not credentials.add_identity_credential("frontend_condortoken", condortoken_file):
+            entry.log.warning(
```
Comment (mambelli) [inline factory/tools/manual_glidein_submit.py:217]: `entry` is not defined. I think it should be `logging.warning(`

**Review by mambelli — CHANGES_REQUESTED** [review]

**Comment by mambelli** [comment]
Please rebase the PR to avoid conflicts.
And the CHANGELOG.md first section usually is copied from the bottom template and titled "Changes Since Last Release" unless a release is cut.

**Comment by mmascher** [comment]
Rebased and conflicts resolved

**Review by mambelli — APPROVED** [review]

**Comment by mambelli** [comment]
Did a selective rebase to eliminate the pre-commit changes and fixed the CHANGELOG (additions were in old releases)

**Pull request merged**